05-assistive-chatbot/test/test_chatbot_api.py

```diff
@@ -0,0 +1,19 @@
+import json
+import unittest
+from chatbot_api import app
+from fastapi.testclient import TestClient
+
+
+# logger = logging.getLogger(f"chatbot.chatbot_api")
+
+client = TestClient(app)
```

Comment:
Should we make this a fixture? https://docs.pytest.org/en/6.2.x/fixture.html#quick-example